docs/help/FAQ/10_Password_Complexity.rst

```diff
@@ -17,17 +17,29 @@ The default complexity is enforced in both :term:`PAM` and :term:`LDAP`.
 Complexity Rules
 ----------------
 
-  * 14 Characters or greater
+  * 15 Characters or greater
   * 1 Upper case letter
   * 1 Lower case letter
   * 1 Number
   * 1 Special character
   * No more than 2 repetitions of the same character
+    * OK: ``aab``
+    * BAD: ``aaa``
+
+  * No more than 3 repetitions of a character from the same character class
+    * OK: ``abc``
+    * BAD: ``abcd``
+
   * No more than 4 characters in a monotonic character sequence
+    * OK: ``1b2c3d``
+    * BAD: ``aBcDe``
```

Comment:
man page shown '12345' and 'fedcb' as examples.  Do we want a descending sequence as an example?